osquery/tables/other/tests/augeas_tests.cpp

```diff
@@ -0,0 +1,103 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+#include <gtest/gtest.h>
+
+#include <osquery/sql.h>
+
+namespace osquery {
+
+namespace tables {
+
+class AugeasTests : public testing::Test {};
+
+TEST_F(AugeasTests, select_all) {
+  auto results = SQL("select * from augeas");
+  ASSERT_TRUE(results.getStatus().ok()) << results.getStatus().what();
+  ASSERT_GT(results.rows().size(), 0U);
+}
+
+TEST_F(AugeasTests, select_all_with_limit) {
+  auto results = SQL("select * from augeas limit 5");
+  ASSERT_TRUE(results.getStatus().ok()) << results.getStatus().what();
+  ASSERT_EQ(results.rows().size(), 5U);
+}
+
+TEST_F(AugeasTests, select_hosts_by_path_expression) {
+  auto results = SQL("select * from augeas where path='/files/etc/hosts'");
+  ASSERT_TRUE(results.getStatus().ok()) << results.getStatus().what();
+  ASSERT_EQ(results.rows().size(), 1U);
+  ASSERT_EQ(results.rows()[0].at("path"), "/files/etc/hosts");
+  ASSERT_EQ(results.rows()[0].at("filename"), "/etc/hosts");
+  ASSERT_EQ(results.rows()[0].at("label"), "hosts");
+  ASSERT_TRUE(results.rows()[0].at("value").empty())
+      << "Value is not empty. Got " << results.rows()[0].at("value")
+      << "instead";
+}
+
+TEST_F(AugeasTests, select_etc_by_path_expression) {
+  auto results = SQL("select * from augeas where path='/files/etc'");
+  ASSERT_TRUE(results.getStatus().ok()) << results.getStatus().what();
```

Comment:
No need to repeat assertions, please make each test unit only assert or expect the test specifics.